nextcord/interactions.py

```diff
@@ -626,16 +633,25 @@ async def send_message(
         }
 
         if embed is not MISSING and embeds is not MISSING:
-            raise TypeError('cannot mix embed and embeds keyword arguments')
+            raise TypeError('Cannot mix embed and embeds keyword arguments')
 
         if embed is not MISSING:
             embeds = [embed]
 
         if embeds:
             if len(embeds) > 10:
-                raise ValueError('embeds cannot exceed maximum of 10 elements')
+                raise ValueError('Embeds cannot exceed maximum of 10 elements')
```

Comment:
@TAG-Epic Should I remove this existing exception from embeds and let it throw the HTTPException instead?

Reply:
Yeah!